README.md

```diff
@@ -1,11 +1,68 @@
 # mobile-client-json
-Json Schema Draft 7 documentation for the JSON model objects used to build the iOS and Android Assessment Models.
+
+Json Schema Draft 7 documentation for the JSON model objects used to build the iOS and Android 
+Assessment Models. The [AssessmentModel][1] library used by all internally developed assessments 
+considers an assessment to be any measurement, including surveys, that can be run on a mobile
+device, provided that the library dependencies for a given app are set up to include it. 
+This repoitory includes the schemas used to define:
+
+- [Result Archives](#result-archives): The output files for a single run of an assessment.
+- [Building a Survey](#building-a-survey): The [AssessmentConfig][2] for describing all the questions and branching logic for running a survey.
+
+[1]: <https://github.com/BridgeDigitalHealth/AssessmentModelKMM> "AssessmentModelKMM"
+[2]: <https://developer.sagebridge.org/model-browser.html#AssessmentConfig> "AssessmentConfig"
+
+## Result Archives
+
+Each successfully completed run of an assessment (including surveys) builds and uploads
+an archive from the mobile device to Bridge services. The archive will include all the 
+files generated by the given run of the assessment.
+
+### Metadata File
+
+All archives include the **required** "metadata.json" file that includes information 
+about the device, assessment schedule, and a manifest of all the other files in the 
+archive. 
+
+The json schema for this file is defined here: [ArchiveMetadata](/schemas/v2/ArchiveMetadata.json)
+
+As well as information about the device, app, and assessment schedule, the file also
+includes a manifest of the other files in the archive with the following properties:
+
+| Key         | Required | Description |
+|-------------|----------|-------------|
+| filename    | true     | The filename of the archive object. This should be unique within the manifest |
+| timestamp   | true     | The file creation date |
+| contentType | false    | The content type of the file |
+| jsonSchema  | false    | The uri for the json schema if the content type is 'application/json'. |
+
+Note: There are other fields in the manifest as described by the json schema, but these 
+are the fields that are most commonly asked about. syoung 09/27/2023
```

Comment:
https://xkcd.com/1179/
I don't actually feel all that strongly about it but it amused me to link this here